src/cli/cli_app.js

```diff
@@ -112,43 +112,27 @@ class CLI {
         console.log(addrs);
         console.log('>------------------------<');
       },
-      'getOutConnections': ()=>{
-        const cons = this._node.getAllOutboundHandshakes();
-        console.log('---> outbound connections <---');
-        cons.forEach((con)=>{
-          console.log(con.id.toB58String());
-        });
-        console.log('>------------------------<');
-      },
-      'getInConnections': ()=>{
-        const cons = this._node.getAllInboundHandshakes();
-        console.log('---> inbound connections <---');
-        cons.forEach((con)=>{
-          console.log(con.id.toB58String());
-        });
-        console.log('>------------------------<');
-      },
       'peerBank': () =>{
         const peers = this._node.getAllPeerBank();
         console.log('peer bank: ');
-        for (let k=0; k<peers.lentgh; k++) {
+        for (let k=0; k<peers.length; k++) {
           console.log(k);
         }
         console.log('>------------------------<');
       },
-      'discover': () =>{
-        this._node.tryConsistentDiscovery();
-      },
-      'inCount': () =>{
-        const cons = this._node.getAllInboundHandshakes();
-        console.log('---> inbound connections <---');
-        console.log(cons.length);
+      'getConnectedPeers': () =>{
+        let peers = this._node.getConnectedPeers();
```

Comment:
Using `const` instead of `let` can catch a lot of bugs.  
(Not needed here tough)